packages/playground/src/portal/twin_view.vue

```diff
@@ -36,7 +36,9 @@
               <v-divider></v-divider>
               <v-list-item>
                 {{ profileManager.profile?.address }}
-                <v-icon style="padding-left: 3rem" @click="copy(profileManager.profile?.address as string)"
+                <v-icon
```

Comment:
Can you make the icon at the end of the row instead?

Reply:
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/d0ff0167-2961-4752-9165-a1d68445a604)